app/src/sharedTest/java/org/oppia/android/app/faq/FAQListFragmentTest.kt

```diff
@@ -107,7 +108,9 @@ class FAQListFragmentTest {
   @Test
   fun openFAQListActivity_selectFAQQuestion_opensFAQSingleActivity() {
     launch(FAQListActivity::class.java).use {
-      onView(atPosition(R.id.faq_fragment_recycler_view, 1)).perform(click())
+      onView(atPosition(
+        recyclerViewId = R.id.faq_fragment_recycler_view,
+        position = 1)).perform(click())
```

Comment:
Formatting is incorrect. There should be a correct way to decide when to go to next and when to continue in same line.
Maybe try using reformatting shortcut:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
@ksog66 Can you reply this comment so that I can know whether you applied these changes or faced any issues.
You can write as simple as "Done".

Reply:
@rt4914 Done